### PR TITLE
Add support for foxglove schema names from IDL schemas

### PR DESCRIPTION
**Pull request by defunctzombie**
**User-Facing Changes**
TBD

**Description**
Add support for recognizing well-known schema names which adhere to the upcoming omgidl spec: https://github.com/foxglove/mcap/pull/894

### CODE REVIEW

**Comment by defunctzombie** [comment]
@jtbandes @snosenzo I wasn't sure what to put for user-facing changes since maybe we want to hold the official announcement until more of the IDL support lands and we publish versions of the schemas in IDL?

**Review by jtbandes — COMMENTED** [review]
Missed some?
https://github.com/foxglove/studio/blob/6ed4b06afc6f1054542bbf5c809350f47c2324cb/packages/studio-base/src/panels/Log/index.tsx#L39
https://github.com/foxglove/studio/blob/6ed4b06afc6f1054542bbf5c809350f47c2324cb/packages/studio-base/src/panels/ThreeDeeRender/renderables/ImageMode/annotations/normalizeAnnotations.ts

**Comment by jtbandes** [comment]
I believe this other file was moved directly from the old image panel without modification.

File: packages/studio-base/src/panels/Map/support.ts
Comment (jtbandes) [inline packages/studio-base/src/panels/Map/support.ts:1]: Missed one in isGeoJSONMessage in this file

**Review by snosenzo — APPROVED** [review]

**Pull request merged**